web/pwa/src/assets/doc/cards.md

```diff
@@ -1 +1,44 @@
-On the "Cards" tab you can manage your virtual storage cards.
+# Virtual Card Manager
+
+**The 'Cards' tab allows you to manage your virtual storage cards.**
+
+
+## Background
+ Many Palm OS devices featured SD card expansion ports. Adding an SD card could be used for many purposes, such as storing addition programs, or storing large files or databases or multimedia. Some software was even distributed on SD cards.
+
+CloudPilot allows you to connect virtual storage cards to your emulated device.
+
+## Managing Cards
+### Adding a card
+ Tap the plus button in the top right corner to add a card. You can either create a blank card, or you can import an existing image file into CloudPilot.
+ 
+ **Importing a card**: Imported cards should be provided in a binary .img file format. 
+ 
+ **Creating a blank card**: If you choose to create a card, you will be asked to configure the size of the virtual card. *Note: the size of the card currently cannot be changed after creation.*
+
+### Exporting a card
+If you wish to export an image of a current card in CloudPilot, right click on the card in the list, and choose 'Save'. An image of the card will then download.
+
+### Deleting a card
+To delete a card, right click on the card in the list, and choose 'Delete'.
+
+### Renaming a card
+To rename a card, right click on the card in the list, and then choose 'Edit'.
+
+### Checking card filesystem
+To verify a card is not corrupted, right click on the card in the list, and choose 'Check'.
```

Comment:
This also allows to repair any filesystem errors. Note that this is not usually necessary to do manually, CP will automatically check cards that have been ejected before using them again.

Some original SD cards (and their images) were produced with errors, and some (the DRM of Encyclopeadia Britannica) will even stop working if they are fixed. This is where the "Automatically check card" option comes into play: it is on by default, but you can disable it in order to stop CP bickering about file system errors 😏 

Formatting cards in PalmOS can also produce file system errors in some cases.